src/components/Game.js

```diff
@@ -13,7 +19,7 @@ export const GameView = () => {
         {game.isDead ? (
           <GameOver />
         ) : (
-          <button onClick={changeTurn}>Attack</button>
```

Comment:
Can we not have both? I'm not sure how it works. but If there is going to be an 'Attack' button on the screen, surely it will be tempting to click on it. But I see no harm in also activating it with the Spacebar.

Reply:
yeah I think both would be best